cudax/include/cuda/experimental/__async/write_env.cuh

```diff
@@ -70,7 +70,7 @@ public:
   /// @brief Wraps one sender in another that modifies the execution
   /// environment by merging in the environment specified.
   template <class _Sndr, class _Env>
-  _CCCL_HOST_DEVICE _CUDAX_ALWAYS_INLINE constexpr auto operator()(_Sndr, _Env) const //
+  _CCCL_HOST_DEVICE _CUDAX_HIDDEN_INLINE constexpr auto operator()(_Sndr, _Env) const //
```

Comment:
In a similar vein, I havent seen this macro being used without `_CCCL_HOST_DEVICE` do we want to add that?

Reply:
that's not unreasonable. `_LIBCUDACXX_HIDE_FROM_ABI` does that too. i'll take the suggestion, thanks.